packages/ui-extensions/src/surfaces/admin/components/Link/examples/basic-link.example.ts

```diff
@@ -0,0 +1,17 @@
+import {
+  extension,
+  Link,
+} from '@shopify/ui-extensions/admin';
+
+export default extension(
+  'Playground',
+  (root) => {
+    const link = root.createComponent(
+      Link,
+      {href: 'https://www.shopify.ca/climate/sustainability-fund'},
```

Comment:
Should we start documenting the [custom protocols we agreed](https://github.com/Shopify/ui-api-design/discussions/20#discussioncomment-6029688) on in these examples? It's a pretty common need for extensions in the Admin to link to other Admin pages or to the App

Reply:
Since this wasn't persisted to the API alignment I wasn't sure it was finalized. I'll add examples with the custom protocols then. On that note, we'll need to add those to our implementation as well.